server/methods/accounts/accounts.app-test.js

```diff
@@ -495,29 +495,32 @@ describe("Account Meteor method ", function () {
       sandbox.stub(Reaction, "hasPermission", () => false);
       const createUserSpy = sandbox.spy(MeteorAccount, "createUser");
       // create user
-      expect(() => Meteor.call("accounts/inviteShopMember", shopId,
-        fakeUser.emails[0].address,
-        fakeUser.profile.addressBook[0].fullName)).to.throw(Meteor.Error, /Access denied/);
+      expect(() =>
+        Meteor.call("accounts/inviteShopMember", {
+          shopId,
+          groupId: Random.id(),
+          email: fakeUser.emails[0].address,
+          name: fakeUser.profile.addressBook[0].fullName
+        })
+      ).to.throw(Meteor.Error, /Access denied/);
       // expect that createUser shouldnt have run
       expect(createUserSpy).to.not.have.been.called;
-      // expect(createUserSpy).to.not.have.been.called.with({
-      //   username: fakeUser.profile.addressBook[0].fullName
-      // });
     });
 
     it("should let a Owner invite a user to the shop", function (done) {
       this.timeout(20000);
       this.retries(3);
       sandbox.stub(Reaction, "hasPermission", () => true);
-      // TODO checking this is failing, even though we can see it happening in the log.
-      // spyOn(Email, "send");
-      expect(function () {
-        return Meteor.call("accounts/inviteShopMember",
+      // TODO: Need to udpate this test to properly check the account created
+      // there's currently an error with Media branding asset when trying to do that
```

Comment:
Not a blocker, but let's chat about why you think this is happening. Maybe next week sometime we can pair on tests together for a bit and see if we can knock this out.